### PR TITLE
Dildos & Tanks

### PR DESCRIPTION
**Pull request by AliiTea**
Adds a new Dildo

Changes prices for tanks to be realistic and makes them hidden unless console is emagged.

### CODE REVIEW

File: tgstation.dme [hunk 1]
```diff
@@ -6,6 +6,78 @@
```
Comment (AliiTea) [inline tgstation.dme:6]: No fucking clue why this happens EVERYTIME.... please fuck off dumb code..

**Comment by Nokington** [comment]
Dildos are dumb and should be removed

**Comment by AliiTea** [comment]
You are dumb and should be removed :^)

**Comment by ghost** [comment]
wowe

**Comment by ghost** [comment]
i agree with @Memendia, also please stop setting `FILE_DIR` turn that off.

**Comment by AliiTea** [comment]
@Chnkr How to do this I am an idiot...

**Comment by ghost** [comment]
Go to `build` -> `Preferences` -> untick `Automatically set FILE_DIR`

**Comment by AliiTea** [comment]
Why thank you.. now I shall go back to hating you.

**Pull request closed without merging**